### PR TITLE
Enhance interop example scripts

### LINKED ISSUE #9097: Generalize scripts in interop example
### Description

The example script should be generalized in a way to make it easier to adjust them for using them to register a custom sidechain.

Descriptive code comments should be added as well.

### Motivation

These scripts shall be used in the guide https://liskhq.github.io/lisk-docs/beta/build-blockchain/register-sidechain.html

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #9097

### How was it solved?

- generalized existing scripts to be used for registering own custom sidechains
- Added descriptive code comments providing more context
-

### CODE REVIEW

File: examples/interop/pos-sidechain-example-one/src/app/modules/hello/schema.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	properties: {
+		reactionType: {
+			dataType: 'uint32',
+			fieldNumber: 1,
+		},
+		/**
+		 * ID of the message.
+		 */
+		helloMessageID: {
+			dataType: 'bytes',
```
Comment (ishantiw) [inline examples/interop/pos-sidechain-example-one/src/app/modules/hello/schema.ts:99]: I think this should be string type if you want to keep everywhere as string

File: examples/interop/pos-sidechain-example-one/src/app/modules/hello/schema.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+			fieldNumber: 3,
+			minLength: 0,
+			maxLength: 64,
+		},
+	},
+};
+
+export interface CCReactMessageParams {
+	reactionType: number;
+	helloMessageID: Buffer;
```
Comment (ishantiw) [inline examples/interop/pos-sidechain-example-one/src/app/modules/hello/schema.ts:114]:
```suggestion
	helloMessageID: string;
```

File: examples/interop/pos-sidechain-example-one/src/app/modules/hello/schema.ts (hunk 1 above)
Comment (Tschakki) [reply examples/interop/pos-sidechain-example-one/src/app/modules/hello/schema.ts:99]:
```suggestion
		helloMessageID: {
			dataType: 'string',
```

**Review by ishantiw — APPROVED** [review]
🎉

**Pull request merged**